source/templates/4-Action-Set-VMSS.json

```diff
@@ -81,7 +81,7 @@
                         },
 						{
                             "condition": "[equals(toLower(parameters('type')), toLower('Microsoft.Compute/virtualMachineScaleSets'))]",
-                            "apiVersion": "2018-11-20",
```

Comment:
I think the HybridCompute apiVersion can be updated as well. We should use the same apiVersion across all of them no?